src/webui/utils/capitalize.js

```diff
@@ -0,0 +1,7 @@
+/**
+ * @prettier
+ * @flow
+ */
+
+const capitalize = (str: string) => `${str[0].toUpperCase()}${str.slice(1)}`;
```

Comment:
lodash/capitalise ?

Reply:
@ayusharma you are right...as I usually don't use lodash..I forgot about it...I'm going to change it..thanks for the reminder 😉